modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -9,7 +9,7 @@ You can use the Bare Metal Operator (BMO) to provision bare-metal hosts in a use
 
 [NOTE]
 ====
-To provision bare-metal hosts to the cluster by using the BMO, you must set the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `false`.
+When provisioning new bare-metal hosts to the cluster using the BMO, the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource is set to `false` by default.
```

Comment:
Perhaps "When you provision new bare-metal hosts..." 

OR

Provisioning new bare-metal hosts to the cluster using the BMO sets the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `false` by default.

Otherwise the 'the `spec.externallyProvisioned` specification becomes the object (i.e., doing the provisioning) and it is not really grammatically correct.

Reply:
It's correct that the default is `false`.

It's also the case that a lot of people copy-and-paste the BMH from one of the control plane nodes (for which `externallyProvisioned` is explicitly set to `true`) and modify only the BMC address, and are then surprised when nothing happens.

That's the reason this note was here, along with the explicit callout below.

Reply:
Updated as suggested.